crates/hyperqueue/src/client/task.rs

```diff
@@ -87,13 +87,32 @@ pub async fn output_job_task_info(
     gsettings: &GlobalSettings,
     session: &mut ClientSession,
     job_id_selector: IdSelector,
```

Comment:
Could you please create something like `SingleIdSelector` (and `SingleIdSelectorArg`) that only supports a single ID or Last? So that we have it checked in compile time and don't even offer users to enter invalid argument in CLI.

There are some other commands that take a single ID as e.g. u32. Could you please check if it would make sense to also use `SingleIdSelector` for some of them?